### ISSUE TITLE
[WARNING] [promise.ControlFlow] Detected scheduling of an unchained task

### ISSUE THREAD

**Issue by emailtomit** [gkushang/nemo-window#5]
`await nemo.window.switchTab();`
above line is causing below warning message with nemo 3.0.3

> [WARNING] [promise.ControlFlow] Detected scheduling of an unchained task.
>     When the promise manager is disabled, unchained tasks will not wait for
>     previously scheduled tasks to finish before starting to execute.
>     New task: Task: WebDriver.getAllWindowHandles()
>         at thenableWebDriverProxy.schedule (/Users/node_modules/selenium-webdriver/lib/webdriver.js:807:17)
>         at thenableWebDriverProxy.getAllWindowHandles (/Users/node_modules/selenium-webdriver/lib/webdriver.js:968:17)
>         at handles (/Users/node_modules/nemo-window/lib/window.js:38:17)
>         at Object.switchTab (/Users/node_modules/nemo-window/lib/window.js:62:10)
>         at Object.validateAdvancedSearch (/Users/test/functional/module/test1.js:158:21)
>         at <anonymous>
>     Previous task: Task: WebDriver.getWindowHandle()
>         at thenableWebDriverProxy.schedule (/Users/node_modules/selenium-webdriver/lib/webdriver.js:807:17)
>         at thenableWebDriverProxy.getWindowHandle (/Users/node_modules/selenium-webdriver/lib/webdriver.js:961:17)
>         at handle (/Users/node_modules/nemo-window/lib/window.js:33:17)
>         at Object.switchTab (/Users/node_modules/nemo-window/lib/window.js:58:3)
>         at Object.validateAdvancedSearch (/Users/test/functional/module/test1.js:158:21)
>         at <anonymous>
